### ISSUE TITLE
“PostSpotBatchOrders” this method has a parameter error

### ISSUE THREAD

**Issue by zzyyq** [bitmartexchange/bitmart-go-sdk-api#2]
I try to use this method “PostSpotBatchOrders”, but it has a parameter error.

``` golang
// PostSpotBatchOrders /** Batch New Order(v2) (SIGNED)
func (cloudClient *CloudClient) PostSpotBatchOrders(orderParams [1]Order) (*CloudResponse, error) {
	params := NewParams()
	params["order_params"] = orderParams
	return cloudClient.requestWithParams(POST, API_SPOT_BATCH_ORDERS_URL, params, SIGNED)
}
```
```orderParams [1]Order``` This parameter will prevent the correct order slice from being sent, it can only accept an order array of length one. I think he should be ```orderParams []Order```.

this api is :
``` shell
curl 
 -H 'X-BM-KEY:{{AccessKey}}'
 -H 'X-BM-TIMESTAMP:{{currentTime}}'
 -H 'X-BM-SIGN:{{SIGN}}' 
 -X POST -d '{
    "order_params":[
        {
        "symbol":"BTC_USDT",
        "size":"0.1",
        "price":"8800",
        "side":"buy",
        "type":"limit"
        },
        {
        "symbol":"BTC_USDT",
        "size":"0.1",
        "price":"8800",
        "side":"sell",
        "type":"limit"
        }
    ]
}'
https://api-cloud.bitmart.com/spot/v2/batch_orders
```

**Comment by 11101171** [comment]
Yes, the parameter here should be defined as `orderParams []Order`

```go
func (cloudClient *CloudClient) PostSpotBatchOrders(orderParams []Order) (*CloudResponse, error) {
params := NewParams()
params["order_params"] = orderParams
return cloudClient.requestWithParams(POST, API_SPOT_BATCH_ORDERS_URL, params, SIGNED)
}

```

**Comment by 11101171** [comment]
Fix this BUG in version v1.0.1

**Comment by Romainbonte** [comment]
Thanck you
